exercises/rational-numbers/canonical-data.json

```diff
@@ -422,6 +422,30 @@
           "expected": [1, 1]
         }
       ]
+    },
+    {
+      "uuid": "bd1eab16-7603-4a77-ae4e-23c930fe39cd",
+      "description": "Ordering",
```

Comment:
I don't have a better alternative, but this description is fairly minimal. Maybe something like `Rational numbers can be ordered` or something like that?

Reply:
The name is in the style of the other names, like "absolute value" or "multiplication". Do you think it should be longer anyway?

Reply:
There is some inconsistency in the test naming. I'm fine with keeping it as is therefore, but let's see what others think.